dialogue-target/src/main/java/com/palantir/dialogue/Response.java

```diff
@@ -17,12 +17,13 @@
 package com.palantir.dialogue;
 
 import com.google.common.collect.ImmutableList;
+import java.io.Closeable;
 import java.io.InputStream;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 
-public interface Response {
+public interface Response extends Closeable {
```

Comment:
Would putting a `@MustBeClosed` annotation on this help us catch any other cases?

Reply:
Unfortunately `MustBeClosed` is only valid for methods to require return values to be closed.